azure-core/azure/core/pipeline/__init__.py

```diff
@@ -113,7 +115,7 @@ def __init__(self, http_request, http_response, context):
         self.context = context
 
 
-from .base import Pipeline
+from .base import Pipeline #pylint: disable=wrong-import-position
```

Comment:
I'm not sure this should be disabled - can the import be moved? Or does it need to be here?

Reply:
moving this is causing tests to fail 
`ImportError: cannot import name 'AbstractContextManager' from 'azure.core.pipeline'`